internal/transport/http2_client.go

```diff
@@ -447,7 +453,7 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 		err := t.loopy.run()
 		if err != nil {
 			if logger.V(logLevel) {
-				logger.Errorf("transport: loopyWriter.run returning. Err: %v", err)
+				logger.Errorf("transport: loopyWriter.run returned. Err: %v. Closing connection.", err)
```

Comment:
Not always required, but mostly makes sense to add the existing error string to the end of the new one. Also, could we skip `loopyWriter.run` and replace it with something more readable?